core/src/main/kotlin/plugins/Lifecycle.kt

```diff
@@ -42,8 +44,10 @@ fun Application.configureLifecycle() {
     environment.monitor.subscribe(DatabaseReady) {
         val authorizationService by inject<AuthorizationService>()
 
-        runBlocking(Dispatchers.IO) {
-            syncRoles(authorizationService)
+        thread {
```

Comment:
Maybe getting a bit off-topic, but could you further explain (here, not in the ocmmit message) about the different to Java's `Thread {}` (with upper-case "T") as used e.g. at

https://github.com/eclipse-apoapsis/ort-server/blob/cff36a377033e4c81752cc7e2212a71639beceb1/transport/spi/src/testFixtures/kotlin/Utils.kt#L74

Reply:
`Thread {}` is just calling the `Thread` constructor with a runnable argument and you have to start it on your own. Kotlin's `thread` factory function also starts the thread automatically and offers some configuration options which are not used here.

Reply:
Ok, so I guess that confirms you're intentionally using `thread {}`, not `Thread {}`, here.

And that we should probably align on the former in existing code?

Reply:
Actually, I never thought about using `Thread {}`. And yes, as `thread {}` avoid the start call I would prefer that.